docs/ui/components/home/GridCell.tsx

```diff
@@ -0,0 +1,35 @@
+import { css, SerializedStyles } from '@emotion/react';
+import { borderRadius, theme } from '@expo/styleguide';
+import React from 'react';
+import { Col } from 'react-grid-system';
+
+type GridCellProps = {
+  children: any;
```

Comment:
I think we can pluck the children property from React's `PropsWithChildren<{ ... }>` type, that would validate if the function returns the right values 😄